test/cql-pytest/test_describe.py

```diff
@@ -736,6 +780,59 @@ def test_table_options_quoting(cql, test_keyspace):
 
 ### =========================== UTILITY FUNCTIONS =============================
 
+# Create random body of UDT using all UDTs from `udts`.
+# The functions tries to create complex field types,
+# by mixing lists, tuples, sets, etc...
+def create_udt_body(udts):
+    two_udts_type_probability = 0.3
+    nest_probability = 0.1
+
+    one_udt_type = lambda u: random.choice([
+        f"frozen<{u}>",
+        f"frozen<list<{u}>>",
+        f"list<frozen<{u}>>",
+        f"frozen<set<{u}>>",
+        f"set<frozen<{u}>>",
+        f"tuple<frozen<{u}>, int>",
+        f"frozen<tuple<{u}, int>>",
+        f"map<frozen<{u}>, int>",
+        f"map<int, frozen<{u}>>",
+        f"frozen<map<{u}, int>>",
+        f"frozen<map<int, {u}>>",
+    ])
+    two_udts_type = lambda u, w: random.choice([
+        f"frozen<tuple<{u}, {w}>>",
+        f"tuple<frozen<{u}>, frozen<{w}>>",
+        f"frozen<map<{u}, {w}>>",
+        f"map<frozen<{u}>, frozen<{w}>>"
+    ])
+    
+    fields = []
+    current_field_name = 'a'
+    while udts:
+        # Select to create one_udt_type or two_udts_type
+        two_udts = False
+        if len(udts) > 1 and random.random() < two_udts_type_probability:
+            two_udts = True
+        
+        field_type = ""
+        if two_udts: # Create a type based on two UDTs (two_udts_type)
+            u1 = udts.pop()
+            u2 = udts.pop()
+            field_type = two_udts_type(u1, u2)
+        else:        # Create a type based on one UDT (one_udt_type)
+            field_type = one_udt_type(udts.pop())
+        
+        # Maybe create nested type
+        if random.random() < nest_probability:
+            field_type = one_udt_type(field_type)
+
+        fields.append(f"{current_field_name} {field_type}")
+        current_field_name = chr(ord(current_field_name) + 1)
```

Comment:
This is too tricky for my taste and wrong when we pass a list with too many UDTs as the argument. If we get past `z` then `(` is the next character, which is not a valid field name and will result in a very confusing parse error. This becomes a risk it you pass more than 26 UDTs.

It looks like it can actually happen in the test you wrote: for example `test_udt_sorting` decides to make 49 independent UDTs, and then one big UDT which depends on all of them.

Can we use something plain and simple, e.g. `f"f_{field_id}"` where `field_id` is just an integer?